foundation/pom.xml

```diff
@@ -512,7 +512,6 @@
                         <sourceEncoding>${project.build.sourceEncoding}</sourceEncoding>
                         <minimumTokens>100</minimumTokens>
                         <failurePriority>${basepom.pmd.fail-level}</failurePriority>
-                        <rulesets />
```

Comment:
Simply removing this line will lead to this code to be active: https://github.com/apache/maven-plugins/blob/trunk/maven-pmd-plugin/src/main/java/org/apache/maven/plugin/pmd/PmdReport.java#L119-L127

Which actually adds a set of policies that may or may not be desired. The current philosophy is that the plugins are active but don't do anything in foundation. If I were to turn it off and leave that line out, then when adding rule sets, you have to do two things: a) activate the plugin and b) add the ruleset.